package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Girls-n-a-Dude",
+  "version": "1.0.0",
+  "author": "",
+  "license": "MIT",
+  "scripts": {
+     "start": "node src/server.js",
+    "start:watch": "nodemon -q src/server.js"
+  },
```

Comment:
indentation is different for "start" and "start:watch". I also prefer "dev" over start:watch (more clear) but that's preference :smile: